### PR TITLE
Feature/add simple handicap

### PR DESCRIPTION
**Pull request by jjm3x3**
Since Sam joined the league for this season late the idea was thrown around of bringing his current score down a few pegs to account for the fact that he would not have likely had a perfect ranking right up until this most recent week. Upon further though, I figured we could bring the total score down by adding a very simple handicapping mechanism. Here is an attempt to do that.

### CODE REVIEW

**Comment by jjm3x3** [comment]
@internette I just realized that this necessarily doesn't cover what happens if someones handicap would make their score negative. Not sure if it matters, just and interesting thought.

**Review by internette — APPROVED** [review]

File: app/leagueData/AmazingRace_36.tsx [hunk 1]
```diff
@@ -31,6 +31,7 @@ export const CONTESTANT_LEAGUE_DATA = [
     },
     {
         name: "Sam",
-        ranking: ["Ricky Rotandi & Cesar Aldrete", "Rod Gardner & Leticia Gardner", "Yvonne Chavez & Melissa Main", "Derek Williams & Shelisa Williams", "Juan Villa & Shane Bilek", "Sunny Pulver & Bizzy Smith", "Amber Craven & Vinny Cagungun", "Angie Butler & Danny Butler", "Kishori Turner & Karishma Cordero", "Anthony Smith & Bailey Smith", "Michelle Clark & Sean Clark", "Chris Foster & Mary Cardona-Foster", "Maya Mody & Rohan Mody"]
+        ranking: ["Ricky Rotandi & Cesar Aldrete", "Rod Gardner & Leticia Gardner", "Yvonne Chavez & Melissa Main", "Derek Williams & Shelisa Williams", "Juan Villa & Shane Bilek", "Sunny Pulver & Bizzy Smith", "Amber Craven & Vinny Cagungun", "Angie Butler & Danny Butler", "Kishori Turner & Karishma Cordero", "Anthony Smith & Bailey Smith", "Michelle Clark & Sean Clark", "Chris Foster & Mary Cardona-Foster", "Maya Mody & Rohan Mody"],
+        handicap: -50
```
Comment (jjm3x3) [inline app/leagueData/AmazingRace_36.tsx:35]: Going to change this as per Sam's request.
```suggestion
        handicap: -80
```

File: __tests__/models/LeagueStanding.js [hunk 2]
```diff
@@ -147,4 +148,47 @@ describe("addContestantRoundScores", () => {
...
+        expect(sut.rouds).not.toBeNull()
+        expect(sut.rounds.length).toBe(1)
+        expect(sut.rounds[0]).not.toBeNull()
+        expect(sut.rounds[0].contestantRoundData).not.toBeNull()
+        expect(sut.rounds[0].contestantRoundData.length).toBe(1)
+        const resultingContestantRoundData =  sut.rounds[0].contestantRoundData
+        expect(resultingContestantRoundData[0].totalScore).toBe(20)
+    })
+
+    it("Should modify totalScore if hadnicap is not passed", () => {
```
Comment (CAnthonySiever) [inline __tests__/models/LeagueStanding.js:173]:
```suggestion
    it("Should modify totalScore if handicap is not passed", () => {
```

File: __tests__/models/LeagueStanding.js [hunk 3]
```diff
@@ -147,4 +148,47 @@ describe("addContestantRoundScores", () => {
...
+        const expectedContestantName1 = "contestant1"
+        const expectedHandicap = -10
+        const sut = new LeagueStanding()
+
+        // Act
+        sut.addContestantRoundScores(teamList, rounds, expectedContestantName1, expectedHandicap)
+
+        // Assert
+        expect(sut).not.toBeNull()
+        expect(sut.rouds).not.toBeNull()
```
Comment (CAnthonySiever) [inline __tests__/models/LeagueStanding.js:186]:
```suggestion
        expect(sut.rounds).not.toBeNull()
```

**Pull request merged**